qa/rpc-tests/wallet.py

```diff
@@ -263,10 +264,13 @@ def run_test (self):
         self.nodes[0].generate(1)
         sync_blocks(self.nodes)
 
-        #tx should be added to balance because after restarting the nodes tx should be broadcastet
+        # tx should be added to balance because after restarting the nodes tx should be broadcast
         assert_equal(self.nodes[2].getbalance(), Decimal('13.99800000')) #should not be
         assert_equal(self.nodes[2].getbalance("*"), Decimal('13.99800000')) #should not be
 
+        # check integer balances from getbalance
+        assert_equal(self.nodes[2].getbalance("*", 1, False, True), 1399800000) #should not be
```

Comment:
Nonblocking: does anyone know what "should not be?" means in these three comments?

Reply:
Seems like it has been there since the code was added in https://github.com/zcash/zcash/commit/77650cc9f4646787e843b2922730d8cba5af36a1 . I suggest just removing it (on 3 lines).

Reply:
Removed comments at https://github.com/zcash/zcash/pull/4282/commits/76cb4e8d0437be9f2843d7affac0f7573605dd12